src/lexer.l

```diff
@@ -509,6 +513,10 @@ STRING_K      ?i:string
 % /* These PSL tokens must be scanned in VHDL mode for look-ahead */
 "->"                     { TOKEN(tIFIMPL); }
 "<->"                    { TOKEN(tIFFIMPL); }
+"[*"                     { TOKEN(tTIMESRPT); }
+"[+]"                    { TOKEN(tPLUSRPT); }
+"[="                     { TOKEN(tGOTORPT); }
+"[->"                    { TOKEN(tARROWRPT); }
```

Comment:
It seems weird that these are tokens rather than separate `[` and `->` but the PSL spec seems to define it this way so ok.

Reply:
Yup, I had similar feeling. Due to similar reason I kept `[[` as two separate tokens (not mentioned in LRM as token), but no examples in LRM show space between these two. I guess it needs some experimenting with other simulators and mimicing the behavior.